apps/climatemappedafrica/src/lib/data/blockify/hero.js

```diff
@@ -7,15 +7,12 @@ import {
  * This function will be called even when HURUmap is disabled.
  * @see @/climatemappedafrica/lib/data/common/index.js
  *
- * TODO(koech): Handle the case when hurumap?.enabled is undefined/false
- *              Should we hide the map?
  */
 export default async function hero(block, _api, _context, { hurumap }) {
   const {
     profilePage,
     rootGeography: { center, code, hasData: pinRootGeography },
   } = hurumap;
```

Comment:
Lets just make sure `hurumap` itself is not `undefined`.

Reply:
@kilemensi [This line](https://github.com/CodeForAfrica/ui/blob/008da14a7781af4fffa59861c1221c27c61e1ed5/apps/climatemappedafrica/src/lib/data/common/index.js#L136-L140) says it can't be `undefined`